backend/src/main/java/com/capstone/moneytree/controller/UserController.java

```diff
@@ -123,10 +124,10 @@ public User login(@RequestBody User credentials) throws CredentialNotFoundExcept
     */
 
    @PostMapping("/{userId}/follow/{userToFollowId}")
-   ResponseEntity<Long> followUser(@PathVariable Long userId, @PathVariable Long userToFollowId) {
-      Long id = this.userService.followUser(userId, userToFollowId);
+   ResponseEntity<String> followUser(@PathVariable Long userId, @PathVariable Long userToFollowId) {
+      String res = this.userService.followUser(userId, userToFollowId);
```

Comment:
What's the motivation with returning a String as a response with "userX is following userY"? Is it because the frontend needs to know who the current user is following explicitely? Normally, we only return relevant information in an object if needed in a response and not a string. Just curious.

Reply:
Honestly I did not know if it's not good practice, I will revert it back to what it was before in my next commit.

Reply:
Before, it was only giving the user ID that it followed. Maybe giving the name instead would be useful or even the whole user object. Not sure what the frontend needs.